### ISSUE TITLE
Add buttons, rich responses to widget for better Rasa integration

### ISSUE THREAD

**Issue by pixpack** [chatwoot/chatwoot#1486]
**Is your feature request related to a problem? Please describe.**
Currently, the widget can't display rich responses from Rasa. Especially buttons.

**Describe the solution you'd like**
Add the possibility to attach buttons to a response as it's done in [Rasa Webchat](https://github.com/botfront/rasa-webchat).

**Describe alternatives you've considered**
Using Rasa Webchat as a widget for Chatwoot or integrating a Rasa chatbot with Chatwoot in a different way than in Chatwoots instructional video.

**Comment by pixpack** [comment]
I've dug into the codebase and I actually found different types of rich messages that are supported. However I think that the documentation on this is insufficient. Detailed explanation on how to send messages with a different `content_type` should be added to the documentation or api reference.

I've also tested the `form` and `input_select` content types and it works well in the widget, but user input isn't shown in chatwoot. 
Here is the payload of my api call:
`{
"content": "Example text",
"message_type": "outgoing",
"private": false,
"content_type": "input_select",
"content_attributes": { 
 "items": [
    {
      "title": "Option 1",
      "value": "option_1"
    },
    {
      "title": "Option 2",
      "value": "option_2"
    }
  ]}
}`
Any help would be appreciated.
